pkgs/applications/terminal-emulators/tym/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchurl, pkg-config, gtk3, vte, lua5_3, pcre2 }:
+
+stdenv.mkDerivation rec {
+  pname = "tym";
+  version = "3.3.0";
+
+  src = fetchurl {
```

Comment:
Is it possible to use `fetchFromGitHub` here instead?

Reply:
I think it's possible, I tried it but I wasn't sure how to make it works, but now I think I can fix this. Should I send a new commit or just force amend and force push?

Reply:
amending is fine :)

Reply:
Well, I think they don't have the make files in the repository, just in the releases, I don't know how to generate the necessary makefiles yet, could you help me?

Reply:
It has `configure.ac`, so I think just adding `autoreconfHook` to `nativeBuildInputs` will solve

Reply:
Done, thank you @applePrincess and @onny